File: C++/PerformanceTest.cpp
/*
todo 更改为多线程
*/

#include <iostream>
using namespace std;

#define MIN 1
#define MAX 100000

int main() {
    long int i;
    long int j;
    long int total;
    for(i = MIN; i<=MAX ; i++ ) {
        for (j= MIN;j<= MAX;j++){
            total += (i+j);
        }
    }
    cout  << total << endl;
    return total;
}

File: Java/PerformanceTest/src/cn/paul/PerformanceTest/PerformanceTest.java
package cn.paul.PerformanceTest;

/**
 * Created by lfp on 2021/2/10.
 */

import java.lang.System;

public class PerformanceTest {
    public static void main(String[] args)
    {
        int MIN=1;
        int MAX=100000;
        int i,j;
        long  total = 0;
        for(i=MIN;i<=MAX;i++){
            for(j=MIN;j<=MAX;j++) {
                total += (i+j);
            }
        }
        System.out.println(total);
    }
}




